examples/ring/vOLE/main.go

```diff
@@ -0,0 +1,406 @@
+package main
```

Comment:
There needs to be an extensive godoc that describes this example and provides links to the paper for further reference.